### ISSUE TITLE
IANA Link Relations registration: Link relation definition

### ISSUE THREAD

**Issue by Kevsy** [ietf-wg-httpapi/api-catalog#18]
From IANA expert review comments:

"(1) Regarding the link relation definition "The link target identifies a catalog of APIs published by the owner of the link context." 

Since the only relationship between link target and link context is this 'ownership' I think it should be more precise what 'owner' means - I do not find any definition of owner in the draft. Can you express that in a well defined term?

In addition, is there any other relationship beyond 'same owner' between link target and context?"

**Comment by Kevsy** [comment]
Using 'link context' was my mistake - I should have said 'link target'. So changed to:

' Description:  The link target identifies a catalog of the APIs published by the owner of the link target domain.'

_updated in local editor's draft_

**Comment by Kevsy** [comment]
Issue reopened: the revised text in draft -06 removed the word 'context' from the IANA description, resulting in this [discussion](https://mailarchive.ietf.org/arch/msg/httpapi/SMaLGQIe6RCPO353DGjQNqvdOvc/) on mailing list:

```
>> In version -06, the registration's "description" field has been
>> updated to change "context" to "target domain" (see Section 7.2):

This seems problematic, in that it allows any link on the Internet to assert 
an API catalogue for a given URI. At the very least some security considerations 
about consuming such an assertion should be outlined.

Furthermore, it's a deviation from the Web Linking specification, which 
defines a link as involving a context, a target, and a relation type. The relation 
type definition can't modify this relationship. As such I suspect that this 
specification isn't defining a link relation per se, it's defining _something else_.
```
Proposed fix is to align the section 7.2 description with the text in section 3.2, 
to ensure compliance with the Web Linking specification:

"Description: The link target identifies a resource that represents a list of APIs 
available from the Publisher of the context resource."

**Comment by Kevsy** [comment]
fixed in https://github.com/ietf-wg-httpapi/api-catalog/commit/a847b22f8a0028323ef0af323d5f1086e08ea8db
